desc/compute/_basis_vectors.py

```diff
@@ -1400,10 +1400,39 @@ def _e_sup_zeta_zz(params, transforms, profiles, data, **kwargs):
         "desc.equilibrium.equilibrium.Equilibrium",
         "desc.geometry.surface.FourierRZToroidalSurface",
     ],
+    aliases=["e_phi"],
+    # Our usual notation implies e_phi = (∂X/∂Φ)|R,Z, but we need to alias e_phi to
+    # e_phi|r,t = (∂X/∂Φ)|ρ,θ for compatibility with older versions of the code.
+    # To instead compute (∂X/∂Φ)|R,Z, look at the quantity "e_phi|R,Z".
 )
-def _e_sub_phi(params, transforms, profiles, data, **kwargs):
-    # dX/dphi at const r,t = dX/dz * dz/dphi = dX/dz / (dphi/dz)
-    data["e_phi"] = (data["e_zeta"].T / data["phi_z"]).T
+def _e_sub_phi_rt(params, transforms, profiles, data, **kwargs):
+    # (∂X/∂Φ)|ρ,θ = (∂X/∂ζ)|ρ,θ / (∂Φ/∂ζ)|ρ,θ
+    data["e_phi|r,t"] = (data["e_zeta"].T / data["phi_z"]).T
+    return data
+
+
+@register_compute_fun(
+    name="e_phi|R,Z",
+    label="\\mathbf{e}_{\\phi} |_{R, Z}",
+    units="m",
+    units_long="meters",
+    description="Tangent vector along increasing cylindrical toroidal angle phi",
```

Comment:
can you maybe make it clear in the description that it is the partial derivative of the position vector evaluated at constant R, Z?